src/Tracy/QueryPanel/templates/panel.phtml

```diff
@@ -1,10 +1,9 @@
 <?php
-
 /** @var mixed[] $parameters */
 /** @var object[] $queries */
 /** @var int $queriesNum */
-/** @var float $totalTime */
 
+/** @var float $totalTime */
```

Comment:
Why that formating change
```
<?php
/** @var mixed[] $parameters */
/** @var object[] $queries */
/** @var int $queriesNum */

/** @var float $totalTime */
use Nettrine\DBAL\Utils\QueryUtils;
```

Reply:
I can fix it- I have turned on auto format code in Netbeans

Reply:
fixed